staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -58,8 +59,31 @@ func HandleCrash(additionalHandlers ...func(interface{})) {
 	}
 }
 
+// HandleCrash simply catches a crash and logs an error. Meant to be called via
+// defer.  Additional context-specific handlers can be provided, and will be
+// called in case of panic.  HandleCrash actually crashes, after calling the
+// handlers and logging the panic message.
+//
+// E.g., you can provide one or more additional handlers for something like shutting down go routines gracefully.
+//
+// The context is used to determine how to log. Should be used instead of HandleCrash.
+func HandleCrashInContext(ctx context.Context, additionalHandlers ...func(context.Context, interface{})) {
```

Comment:
WithContext is common, not InContext

Reply:
True. My thinking was that "handle the error in the context in which it occurred" is a better description of what the function does. Having said that, `HandleWithContext` is also okay.

Reply:
Changed.